src/csvcubedmodels/rdf/dcat.py

```diff
@@ -119,6 +119,54 @@ def __init__(self, uri: str):
         Resource.__init__(self, uri)
         self.rdf_types.add(DCAT.Dataset)
 
+class Distribution(Resource):
+
```

Comment:
Should `Distribution` really inherit from `Resource`? Does the diagram show this?


![The diagram](https://www.w3.org/TR/vocab-dcat-2/images/DCAT-summary-all-attributes.png)

Reply:
It shouldn't inherit from `dcat:Dataset` either. That arrow is showing a `hasA` relationship, not an inheritance relationship